### PR TITLE
Minor modifications

### PR DESCRIPTION
**Pull request by Christina96**

### CODE REVIEW

File: manuscript/11-SoftEngProfPr.txt [hunk 1]
```diff
@@ -233,10 +230,10 @@ Machinery (ACM) και από το διοικητικό συμβούλιο τη
...
 
 
-  * Την καθιέρωση και την διάδοση ενός σώματος γενικά αποδεκτής γνώσης
-  * Την διαπίστευση, την πιστοποίηση και την χορήγηση αδειών
-  * Την διακήρυξη πειθαρχικών μέτρων
-  * Την εξέλιξη του επαγγέλματος μέσω συνεδρίων, εκπαίδευσης και δημοσιεύσεων
+  * Την καθιέρωση και την διάδοση ενός σώματος γενικά αποδεκτής γνώσης,
+  * Την διαπίστευση, την πιστοποίηση και την χορήγηση αδειών,
+  * Την διακήρυξη πειθαρχικών μέτρων,
+  * Την εξέλιξη του επαγγέλματος μέσω συνεδρίων, εκπαίδευσης και δημοσιεύσεων.
```
Comment (dspinellis) [inline manuscript/11-SoftEngProfPr.txt]: I agree with the comma.  But each line should then start with a lowercase letter.

File: manuscript/10-SoftQual.txt [hunk 2]
```diff
@@ -559,11 +559,11 @@ Off-the-Shelf Software) (COTS) και παροχή υπηρεσιών μετά 
...
-  * Δήλωση των στόχων
-  * Συγκεκριμένο προϊόν λογισμικού
-  * Συγκεκριμένο σχέδιο διοίκησης έργου
-  * Λίστα θεμάτων συνδεδεμένη με αυτό το προϊόν
-  * Διαδικασία τεχνικής επιθεώρησης
+  * Δήλωση των στόχων.
+  * Συγκεκριμένο προϊόν λογισμικού.
+  * Συγκεκριμένο σχέδιο διοίκησης έργου.
+  * Λίστα θεμάτων συνδεδεμένη με αυτό το προϊόν.
+  * Διαδικασία τεχνικής επιθεώρησης.
```
Comment (dspinellis) [inline manuscript/10-SoftQual.txt]: For the sake of consistency (see below) and syntactical correctness, please separate with commas and start each line with a lowercase letter.

File: manuscript/5-SoftMainten.txt [hunk 3]
```diff
@@ -134,7 +134,7 @@
   * η διατήρηση του ελέγχου επί των καθημερινών λειτουργιών του λογισμικού
   * η διατήρηση του ελέγχου επί της τροποποίησης του λογισμικού
   * η βελτιστοποίηση των υπαρχόντων λειτουργιών
-  * ο προσδιορισμός απειλών ασφάλειας και η διόρθωση τρωτών σημείων
+  * ο προσδιορισμός απειλών ασφαλείας και η διόρθωση τρωτών σημείων
```
Comment (dspinellis) [inline manuscript/5-SoftMainten.txt]: Also add commas?

**Comment by Christina96** [comment]
Add commas

**Comment by dspinellis** [comment]
Thank you for the swift response!  Please submit the final  pull request as a branch commit that can be cleanly applied (not conflicts).

**Comment by dspinellis** [comment]
Merged, thank you!

**Pull request merged**